### PR TITLE
[zh]fixed a typo for customize-installation-helm (#13462)

### PR DESCRIPTION
**Pull request by Adityayxt**
* fixed a typo for customize-installation-helm

* #apply-the-patch-using-helm-Post-Renderer: Convert uppercase to lowercase

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/setup/additional-setup/customize-installation-helm/index.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ $ helm template istio-ingress istio/gateway --namespace istio-ingress --post-ren
         value: "10"
 {{< /text >}}
 
-### 使用 Helm 应用补丁 Post-Renderer{#apply-the-patch-using-helm-`Post-Renderer`}
+### 使用 Helm 应用补丁 `Post-Renderer`{#apply-the-patch-using-helm-post-renderer}
```
Comment (windsonsea) [inline content/zh/docs/setup/additional-setup/customize-installation-helm/index.md:116]: This has been covered in #13462

**Review by windsonsea — COMMENTED** [review]
#13420 and #13265 made some more changes. Would you like to cover those changes too?

**Comment by Adityayxt** [comment]
> #13420 and #13265 made some more changes. Would you like to cover those changes too?

#13420 changes has been in this pr，and #13265 （content/en/docs/setup/additional-setup/cni/index.md）is not this file

**Review by windsonsea — APPROVED** [review]
Thanks. LGTM.

**Pull request merged**